cli/cmd/plugin/conformance/go.mod

```diff
@@ -2,10 +2,10 @@ module github.com/vmware-tanzu/community-edition/cli/cmd/plugin/conformance
 
 go 1.16
 
+replace github.com/vmware-tanzu/community-edition => ../../../../
```

Comment:
I'm guessing this is just leftover from testing?

Reply:
No, this is necessary to get the plugin (since they are their own separate modules) to use the local code from the repo and not try to download the last release.

Reply:
Hmmm wouldn't we want this to _eventually_ point to a published upstream version of the module? Makes sense to get the _local_ version now since this change isn't in a release, but I'd imagine this eventually would change to:

```go
require (
   github.com/vmware-tanzu/community-edition v0.x.x   // version with local plugin definition
```